src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet"
+      href="./styles/style.css">
+    <link rel="stylesheet"
+      href="./styles/product-card.css">
+    <link rel="stylesheet"
+      href="./styles/stars.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="./styles/style.css">
    <link rel="stylesheet" href="./styles/product-card.css">
    <link rel="stylesheet" href="./styles/stars.css">
```

You have only 2 selectors in each line, so you don't have to put each of them in new line